flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/schema/TableSourceTable.scala

```diff
@@ -65,15 +65,8 @@ class TableSourceTable(
     statistic) {
 
   override def getQualifiedName: util.List[String] = {
-    val names = super.getQualifiedName
     val builder = ImmutableList.builder[String]()
-    builder.addAll(names)
-    if (dynamicOptions.size() != 0) {
```

Comment:
It seems that `dynamicOptions` is never used in this class, should we remove it?

Reply:
Yes, it can be removed.